### ISSUE TITLE
Should align text to the center

### ISSUE THREAD

**Issue by unitehenry** [striven-erp/striven-editor#117]

**Comment by andreepratama27** [comment]
Hi, can i take this one?

**Comment by unitehenry** [comment]
@andreepratama27 Absolutely! You can write this test here: striven-editor/demo/src/test.js

Please reach out for any help.

**Comment by andreepratama27** [comment]
Thanks @unitehenry .

Can i know which text that have to be aligned to center? which file is it?

**Comment by unitehenry** [comment]
If you look at the other tests, many of the use ```se.setcontent()``` to insert some content to apply formatting to.

Let me know if you need more direction.

**Comment by andreepratama27** [comment]
Understood. Let me read some stuff on test, thanks! @unitehenry
